package.json

```diff
@@ -62,7 +61,8 @@
       "env": {
         "description": "Commands to manage your environments, such as orgs and compute environments."
       }
-    }
+    },
+    "flexibleTaxonomy": true
```

Comment:
can't comment because there's no changes - but the `clean:lib` and `postpack` scripts should delete `oclif.lock`